dot/core/messages.go

```diff
@@ -17,29 +17,31 @@
 package core
 
 import (
+	"reflect"
+
 	"github.com/ChainSafe/gossamer/dot/network"
 	"github.com/ChainSafe/gossamer/dot/types"
 	"github.com/ChainSafe/gossamer/lib/transaction"
 )
 
 // HandleTransactionMessage validates each transaction in the message and
 // adds valid transactions to the transaction queue of the BABE session
-func (s *Service) HandleTransactionMessage(msg *network.TransactionMessage) error {
+func (s *Service) HandleTransactionMessage(msg *network.TransactionMessage) (bool, error) {
 	logger.Debug("received TransactionMessage")
 	if !s.isBlockProducer {
-		return nil
+		return false, nil
 	}
```

Comment:
you can remove these lines, we should be validating + propagating tx messages even if we aren't block producers

Reply:
removed.